crates/xmtp-proto/src/gen/mod.rs

```diff
@@ -0,0 +1,25 @@
+// @generated
+pub mod xmtp {
```

Comment:
I feel like a moron for having to handwrite this file myself in the original branch. I felt like a genius at the time for figuring out how to deal with our package names though 😅

Reply:
Oh man, I never would have figured that out on my own.